doc/install.md

```diff
@@ -22,7 +22,7 @@ This creates an `RTMP` endpoint on `127.0.0.1:1935` and an `HLS` media server on
 
 You can serve `RTMP` content into the endpoint using the following command:
 ```
-ffmpeg -re -f lavfi -i testsrc=size=1920x1080:rate=30,format=yuv420p -f lavfi -i sine -threads 1 -c:v libx264 -b:v 4000000 -preset ultrafast -x264-params keyint=30 -strict -2 -c:a aac -f flv rtmp://127.0.0.1:1935/test_signal
+ffmpeg -re -f lavfi -i testsrc=size=1000x1000:rate=30,format=yuv420p -f lavfi -i sine -threads 1 -c:v libx264 -b:v 4000000 -preset ultrafast -x264-params keyint=30 -strict -2 -c:a aac -f flv rtmp://127.0.0.1:1935/test_signal
```

Comment:
I'm curious why this was done? 1080p is a standard resolution...

Reply:
To highlight that resolutions don't need to be standard when you are building your own thing.

Also, streaming unusual formats opens up scope for using this data in different ways which have not yet been thought of.

For example, streaming in portrait is very possible, e.g. 1080x1920, to fit better on a smartphone being held how we all hold them.

Other examples are yet to be defined, but I feel that suggesting "standard" resolutions will serve to limit the scope of what people are able to imagine.

Reply:
Also, devices are coming with different screen sizes, like my phone is 1080 x 2160

Reply:
And finally, the final proposed version suggests `testsrc=size=360x640`, specifically to provoke thinking about video formats for viewing on mobile devices.

![image](https://user-images.githubusercontent.com/59374467/80063781-4f282700-8554-11ea-8332-4266c13968b6.png)

Reply:
> testsrc=size=360x640, specifically to provoke thinking about video formats for viewing on mobile devices.

I have to disagree - when I see 360x640, "mobile device" is not the first thing that comes to mind. Anyway, in the interests of minimizing back-and-forth nitpicking, I'll probably take a last fix up pass over this merge by EOW. Thanks for your work on this!